tools/mask.py

```diff
@@ -38,6 +38,8 @@ def __init__(self, arguments):
         self._input_is_faces = arguments.input_type == "faces"
         self._mask_type = arguments.masker
         self._output_opts = dict(blur_kernel=arguments.blur_kernel, threshold=arguments.threshold)
+        self._output_type = arguments.output_type
+        self._output_full_frame = arguments.full_frame
 
```

Comment:
```py
        self._output_suffix = self._get_output_suffix()
```

Reply:
Then create function:
```py
    def _get_output_suffix(self):
        """ The filename suffix, based on selected output options

        Returns
        -------
        str:
            The suffix to be appended to the output filename
        """
        sfx = "{}_mask_preview_".format(self._mask_type)
        sfx += "face_" if not self._output_full_frame or self._input_is_faces else "frame_"
        sfx += "{}.png".format(self._output_type)
        return sfx
```

Reply:
then in function `_save`

change:
```py
        filename = os.path.join(self._saver.location,
                                "{}_{}_{}_mask_preview.png".format(os.path.splitext(frame)[0],
                                                                   idx,
                                                                   self._mask_type))
```
to:
```py
        filename = os.path.join(self._saver.location, "{}_{}_{}".format(os.path.splitext(frame)[0],
                                                                        idx,
                                                                        self._output_suffix))
```

Reply:
eg:
![image](https://user-images.githubusercontent.com/36920800/69244715-54480b00-0b9d-11ea-88fd-94a3af41876f.png)

Reply:
Done. Thanks for the suggestion.
Definitely makes sense.